markdown/util.py

```diff
@@ -85,19 +85,7 @@
 # Extensions should use "markdown.util.etree" instead of "etree" (or do `from
 # markdown.util import etree`).  Do not import it by yourself.
 
-try:  # pragma: no cover
-    # Is the C implementation of ElementTree available?
-    import xml.etree.cElementTree as etree
-    from xml.etree.ElementTree import Comment
-    # Serializers (including ours) test with non-c Comment
-    etree.test_comment = Comment
-    if etree.VERSION < "1.0.5":
-        raise RuntimeError("cElementTree version 1.0.5 or higher is required.")
-except (ImportError, RuntimeError):  # pragma: no cover
-    # Use the Python implementation of ElementTree?
-    import xml.etree.ElementTree as etree
-    if etree.VERSION < "1.1":
-        raise RuntimeError("ElementTree version 1.1 or higher is required")
+import xml.etree.ElementTree as etree
```

Comment:
This is getting [two flake8 errors][1]. Given the situation, those errors are acceptable. Therefore a comment needs to be added to this line to tell Flake8 to ignore the line: `# noqa`.

[1]: https://travis-ci.org/Python-Markdown/markdown/jobs/644817931#L242